www/utils/markdown.ts

```diff
@@ -1,7 +1,7 @@
-export * as gfm from "https://deno.land/x/gfm@0.1.23/mod.ts";
-import "https://esm.sh/prismjs@1.27.0/components/prism-jsx.js?no-check";
-import "https://esm.sh/prismjs@1.27.0/components/prism-typescript.js?no-check";
-import "https://esm.sh/prismjs@1.27.0/components/prism-tsx.js?no-check";
-import "https://esm.sh/prismjs@1.27.0/components/prism-diff.js?no-check";
+export * as gfm from "https://deno.land/x/gfm@0.1.30/mod.ts";
+import "https://esm.sh/prismjs@1.29.0/components/prism-jsx.js?no-check";
+import "https://esm.sh/prismjs@1.29.0/components/prism-typescript.js?no-check";
+import "https://esm.sh/prismjs@1.29.0/components/prism-tsx.js?no-check";
+import "https://esm.sh/prismjs@1.29.0/components/prism-diff.js?no-check";
 
-export { extract as frontMatter } from "$std/encoding/front_matter.ts";
+export { extract as frontMatter } from "$std/encoding/front_matter/any.ts";
```

Comment:
Do we need `front_matter/any.ts`? I guess `front_matter/yaml.ts` is enough for us